autorally_core/launch/stateEstimator.launch

```diff
@@ -34,14 +34,16 @@
       
       <!--<param name="Gravity" value="-9.79509"/>-->
       
-      <!-- in sumulation use "rosparam set /gps_imu/FixedInitialPose true" -->
+      <!-- To use in sumulation set "rosparam set /gps_imu/FixedInitialPose true" -->
+      <!-- and <param name="InvertY" value="false"/> -->
+      <!-- and <param name="InvertZ" value="false"/> -->
       <param name="FixedInitialPose" value="true"/>
       <param name="initialRoll" value="0"/>
       <param name="intialPitch" value="0"/>
       <param name="initialYaw" value="0"/>
       <param name="InvertX" value="false"/>
-      <param name="InvertY" value="true"/>
-      <param name="InvertZ" value="true"/>
+      <param name="InvertY" value="false"/>
```

Comment:
Inverty and invertz need to be true in the committed code to work on the hardware